<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translator Tool</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <layout class="QVBoxLayout" name="layoutCustomer">
       <item>
        <layout class="QVBoxLayout" name="layoutCustomerAvatar">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutCustomerAvatar">
           <item>
            <spacer name="spacerLeftCustomer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelCustomerAvatar">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="pixmap">
              <pixmap>images/avatar.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacerRightCustomer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelCustomerText">
         <property name="text">
          <string>Customer</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboCustomerLang">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <item>
          <property name="text">
           <string>Japanese (日本)</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>images/jp.png</normaloff>images/jp.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English (English)</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>images/uk.png</normaloff>images/uk.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vietnamese (Việt Nam)</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>images/vn.png</normaloff>images/vn.png</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutMic">
       <item>
        <layout class="QVBoxLayout" name="layoutMicInner">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutMic">
           <item>
            <spacer name="spacerLeftMic">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelMic">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="pixmap">
              <pixmap>images/micro.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacerRightMic">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="buttonMicToggle">
         <property name="text">
          <string>Live Translate: OFF</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutMe">
       <item>
        <layout class="QVBoxLayout" name="layoutMeAvatar">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutMeAvatar">
           <item>
            <spacer name="spacerLeftMe">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelMeAvatar">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="pixmap">
              <pixmap>images/avatar.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacerRightMe">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelMeText">
         <property name="text">
          <string>Me</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboMyLang">
         <item>
          <property name="text">
           <string>Vietnamese (Việt Nam)</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>images/vn.png</normaloff>images/vn.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English (English)</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>images/uk.png</normaloff>images/uk.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Japanese (日本)</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>images/jp.png</normaloff>images/jp.png</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="leftMargin">
      <number>280</number>
     </property>
     <property name="rightMargin">
      <number>280</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonNewConversation">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>NEW Conversation</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QTextEdit" name="textChatBox">
       <property name="placeholderText">
        <string>Meeting chats...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textSummaryBox">
       <property name="placeholderText">
        <string>LLM summary area...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
